src/components/datepicker/calendarMonth.js

```diff
@@ -125,7 +125,7 @@
       var cellText = this.dateLocale.dates[opt_date.getDate()];
 
       if (this.dateUtil.isDateWithinRange(opt_date,
-          this.calendarCtrl.minDate, this.calendarCtrl.maxDate)) {
+          this.calendarCtrl.minDate, this.calendarCtrl.maxDate) && (!angular.isFunction(this.calendarCtrl.isDateEnabled) || this.calendarCtrl.isDateEnabled(opt_date))) {
```

Comment:
Refactor this condition out into a new function `CalendarMonthCtrl.prototype.isDateEnabled`